.github/workflows/initial_CI.yml

```diff
@@ -0,0 +1,40 @@
+name: Django CI
+
+on:
+#   push:
+#     branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 20
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.7
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip 
+        pip install pipenv
+        cd givitsite
+        pipenv sync
+       
+        
+    - name: DB creation
+      run: |
+        sudo apt install postgresql postgresql-contrib postgresql-common postgresql-client libpq-dev
+        sudo systemctl start postgresql
+        sudo -u postgres createdb givit 
+        sudo -u postgres psql -c "ALTER ROLE postgres WITH PASSWORD 'givit';" 
```

Comment:
Why not using the bootstrap.sh script instead of all those duplicate commands? What if your bootstrap.sh script changes? Would you need to update this file as well? Isn't it a redundancy?

Reply:
@lioramilbaum the problem is that our `bootstrap.sh` include some command that relevant for vagrant up operation
(like `runserver`..)

Reply:
Why isn't it relevant for CI?

Reply:
@lioramilbaum maybe it is relevant...
but the problem is that this `bootstrap.sh` written to Vagrant (with operations that relevant just for vagrant).

Reply:
Then you should amend the script such it can run both in a dev environment (via vagrant) and in a ci environment. 
Use if else statements.